### PR TITLE
Change the default password in docs for qpc server

### PR DESCRIPTION
**Pull request by kholdaway**
Change the default quipucords server password

### CODE REVIEW

**Review by abaiken — APPROVED** [review]

File: asciidoc-conversion/modules/proc-logging-into-qpc.adoc [hunk 1]
```diff
@@ -12,6 +12,6 @@ After the connection between the :product-title_cmd: command line tool and the :
 # :product-title_cmd: server login
 ----
 
-. Enter the server user name and password at the prompts. The default login is admin and the password is '+pass+'.
+. Enter the server user name and password at the prompts. The default login is admin and the password is '+qpcpassw0rd+'.
```
Comment (jlprevatt) [inline asciidoc-conversion/modules/proc-logging-into-qpc.adoc]: change: "default login" to "default user name" 
change "and the password is" to "and the default password is"
change: tagging around "admin" to match tagging around "qpcpassw0rd" (to '+admin+')

File: source/cli_login.rst [hunk 2]
```diff
@@ -9,6 +9,6 @@ After the connection between the qpc command line tool and the Quipucords server
 
     # qpc server login
 
-2. Enter the server user name and password at the prompts.  The default login is ``admin`` and the password is ``pass``.
+2. Enter the server user name and password at the prompts.  The default login is ``admin`` and the password is ``qpcpassw0rd``.
```
Comment (jlprevatt) [inline source/cli_login.rst]: change: "default login" to "default user name" 
change "and the password is" to "and the default password is"

File: source/install_scripted.rst [hunk 3]
```diff
@@ -127,6 +127,12 @@ Options:
     - Contains the fully qualified path to the installation directory for the Quipucords server. Defaults to ``~/quipucords/``.
  - **server_port**
     - Contains the port number for the Quipucords server. Defaults to ``9443``.
+ - **server_username**
```
Comment (jlprevatt) [inline source/install_scripted.rst]: add tagging around the values here, so
``admin``
``qpcpassw0rd``
``admin@example.com``

File: asciidoc-conversion/modules/proc-logging-into-qpc.adoc (hunk 1 above)
Comment (kholdaway) [reply asciidoc-conversion/modules/proc-logging-into-qpc.adoc]: 👍

File: source/cli_login.rst (hunk 2 above)
Comment (kholdaway) [reply source/cli_login.rst]: 👍

File: source/install_scripted.rst (hunk 3 above)
Comment (kholdaway) [reply source/install_scripted.rst]: 👍

**Review by jlprevatt-zz — APPROVED** [review]

**Pull request merged**